### ISSUE TITLE
Release new version with fixes

### ISSUE THREAD

**Issue by PhilippSelenium** [bcoe/secure-smtpd#37]
Is there any chance we can have a new release and pypi package including the fixes since 2015?

**Comment by rayzorben** [comment]
Agreed. In the meantime you can simply do `sudo pip install git+https://github.com/bcoe/secure-smtpd.git`
